src/mumble/TextToSpeech_unix.cpp

```diff
@@ -60,6 +60,22 @@ TextToSpeechPrivate::TextToSpeechPrivate() {
 	if (! spd) {
 		qWarning("TextToSpeech: Failed to contact speech dispatcher.");
 	} else {
+		QString lang;
+		if (!g.s.qsTTSLanguage.isEmpty()) {
+			lang = g.s.qsTTSLanguage;
+		} else if (!g.s.qsLanguage.isEmpty()) {
+			QLocale locale(g.s.qsLanguage);
+			lang = locale.bcp47Name();
+		} else {
+			QLocale systemLocale;
+			lang = systemLocale.bcp47Name();
+		}
+		if (!lang.isEmpty()) {
+			if (spd_set_language(spd, lang.toLocal8Bit().constData()) != 0) {
```

Comment:
Do you know where I can find documentation on the speech dispatcher API? Seems surprisingly google resistant.